app/scripts/lib/metrics.js

```diff
@@ -141,7 +141,7 @@ define([
     /**
      * Send the collected data to the backend.
      */
-    flush: function () {
+    flush: function (flushSynchronously) {
```

Comment:
The change from `flushSynchronously` to `async` in _send is a bit confusing. Can we keep the same semantics in both places?